tekton/release-api.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: results-release-api-
+spec:
+  serviceAccountName: tekton-releaser
+  pipelineRef:
+    name: results-release
+  params:
+    - name: revision
+      value: main
+    - name: package
+      value: github.com/tektoncd/results/cmd/api
+    - name: docker_repo
+      value: <TODO>
```

Comment:
was this intentional? Should it be gcr.io/tekton-releases or gcr.io/tekton-releases-nightly?

Reply:
or ko.local?

Reply:
Done! (this will eventually be one of the tekton-releases repos, but for now ko.local works)